extensions/sql-database-projects/src/models/project.ts

```diff
@@ -34,7 +34,7 @@ export class Project implements ISqlProject {
 	private _preDeployScripts: FileProjectEntry[] = [];
 	private _postDeployScripts: FileProjectEntry[] = [];
 	private _noneDeployScripts: FileProjectEntry[] = [];
-	private _isMsbuildSdkStyleProject: boolean = false;
+	private _isSdkStyleProject: boolean = false;
```

Comment:
Might be worth adding a comment here with a link to https://docs.microsoft.com/en-us/dotnet/core/project-sdk/overview so anyone reading the code can understand exactly what this means